src/launch/launch.scss

```diff
@@ -1,3 +1,110 @@
 @import "styles/colors.scss";
+@import "styles/styles.scss";
 .page.launch {
+  display: grid;
+  justify-content: center;
+  grid-gap: 30px;
+  .banner {
+    .banner1 {
+      // Make like a heading
+      @extend .heading;
+      font-size: 4vw;
+      display: grid;
+      justify-content: center;
+    }
+    // TODO: Customize for mobile
+    .banner2 {
+      @extend .subtext;
+      // TODO: Get font from figma
+      font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
+        "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+      display: grid;
+      justify-content: center;
+      text-align: center;
+    }
+  }
+  // Let's define layouts for the launch divs
+  .launch-divs {
+    display: flex;
+    flex-flow: row wrap;
+    flex: 1;
+    width: 100vw;
+    align-items: center;
+    justify-content: space-evenly;
+  }
+  // ** For styling the container holding buttons
+  .div-style {
```

Comment:
I would not use "div-" in the style names, nor in the name of the custom element you created.  better to leave the implementation out of the names, cause it could change

Reply:
okay